.env.example

```
@@ -25,6 +25,9 @@ CACHE__ENABLE_REDIS=false
 # Preencha somente quando CACHE__ENABLE_REDIS=true
 # Para Upstash, use a Redis URL TLS (rediss://), nao a REST URL/token
 CACHE__REDIS_URL=rediss://default:change_me@host:6379
+# Limite por item no Redis para manter o plano pequeno viavel.
+# Mantem o cache focado em respostas curtas e repetidas.
+CACHE__MAX_PAYLOAD_BYTES=32768
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CACHE__MAX_PAYLOAD_BYTES key should go before the CACHE__REDIS_URL key

(UnorderedKey)